doc/en/announce/release-7.0.1.rst

```diff
@@ -0,0 +1,67 @@
+pytest-7.0.1
+=======================================
+
+pytest 7.0.1 has just been released to PyPI.
+
+This is a bug-fix release, being a drop-in replacement. To upgrade::
+
+  pip install --upgrade pytest
+
+The full changelog is available at https://docs.pytest.org/en/stable/changelog.html.
+
+Thanks to all of the contributors to this release:
```

Comment:
Not sure if this is important enough to block the release, but I doubt this is accurate, unless we are deliberately including contributors to pytest 7.0.0 here again or something? Seems more like a bug in our scripts.